src/commons/editingWorkspace/EditingWorkspace.tsx

```diff
@@ -166,22 +167,29 @@ class EditingWorkspace extends React.Component<EditingWorkspaceProps, State> {
     const question: Question = this.state.assessment.questions[questionId];
     const workspaceProps: WorkspaceProps = {
       controlBarProps: this.controlBarProps(questionId),
-      editorProps:
+      editorContainerProps:
         question.type === QuestionTypes.programming
           ? {
+              editorTabs: this.props.editorTabs
+                .map(convertEditorTabStateToProps)
+                .map((editorTabStateProps, index) => {
+                  // TODO: Hardcoded to make use of the first editor tab. Rewrite after editor tabs are added.
+                  //       Specifically, need to update questions such that they can span multiple files.
+                  if (index !== 0) {
+                    return editorTabStateProps;
+                  }
+                  return {
+                    ...editorTabStateProps,
+                    editorValue:
+                      editorTabStateProps.editorValue ||
+                      question.editorValue ||
+                      (question as IProgrammingQuestion).solutionTemplate
+                  };
+                }),
               editorSessionId: '',
-              editorValue:
-                // TODO: Hardcoded to make use of the first editor tab. Rewrite after editor tabs are added.
-                this.props.editorTabs[0].value ||
-                question.editorValue ||
-                (question as IProgrammingQuestion).solutionTemplate,
```

Comment:
Note: the extra `map` is simply to maintain the current behaviour of only replacing the first editor tab.